spec/api_spec.rb

```diff
@@ -49,29 +50,29 @@
   end
 
   describe 'update subscription' do
-    let(:contact) { Maropost::Contact.new(id: nil, email: 'test@test.com') }
-    let(:existing_contact) { Maropost::Contact.new(id: 741_000_000, email: 'test@test.com') }
+    let(:contact) { Maropost::Contact.new(id: nil, email: email) }
+    let(:existing_contact) { Maropost::Contact.new(id: 741_000_000, email: email) }
 
     subject { Maropost::Api.update_subscriptions(contact) }
 
     before(:each) do
       stub_find_maropost_contact(email: contact.email)
       stub_update_maropost_contact(contact_id: existing_contact.id)
       stub_do_not_mail_list_exists(
-        email: 'test@test.com',
+        email: 'test+001@test.com',
         body: read_fixture('do_not_mail_list', 'do_not_mail_not_found.json')
       )
     end
 
     context 'contact exists on maropost' do
       it 'calls update' do
-        stub_do_not_mail_list_create
+        stub_do_not_mail_list_delete
```

Comment:
is this an intended change?

Reply:
It is - the previous behaviour was slightly broken.